projects/angular/src/emphasis/alert/alerts-pager.html

```diff
@@ -6,14 +6,28 @@
 
 <div class="alerts-pager-control">
   <div class="alerts-page-down">
-    <button class="alerts-pager-button" (click)="pageDown()">
-      <cds-icon shape="angle" direction="left" [attr.title]="commonStrings.keys.previous"></cds-icon>
+    <button
+      class="alerts-pager-button"
+      type="button"
+      (click)="pageDown()"
+      [attr.aria-label]="commonStrings.keys.alertPreviousAlertAriaLabel"
+      aria-describedby="current-alert"
+    >
+      <cds-icon shape="angle" direction="left"></cds-icon>
     </button>
   </div>
-  <div class="alerts-pager-text">{{this.multiAlertService.current+1}} / {{this.multiAlertService.count}}</div>
+  <div class="alerts-pager-text" id="alerts-pager" [attr.aria-label]="alertsCurrentPageAriaLabel" aria-current="true">
```

Comment:
I don't think `aria-current="true"` is correct here.

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-current